source/retryable-writes/retryable-writes.rst

```diff
@@ -3,14 +3,14 @@ Retryable Writes
 ================
 
 :Spec Title: Retryable Writes
-:Spec Version: 1.6.1
+:Spec Version: 1.6.2
```

Comment:
This is a patch version bump (typically used for grammar/typo fixes). I was suggesting a minor version bump (i.e. 1.7.0) since this is technically changing driver behavior.

If other folks feel this change is sufficiently insignificant I could go either way, though.

Reply:
Ah whoops I meant to do a minor version bump, I'll update. I agree that a minor version bump is the right way to go.